install.sh

```diff
@@ -269,13 +271,8 @@ verify_init_system() {
     return
   fi
   # UpStart
-  if [ -x /sbin/upstart ]; then
-    INIT_SYSTEM=upstart
-    return
-  fi
-  # Generic initv (also detects upstart in ubuntu 14.04)
   if [ -x /sbin/init ]; then
```

Comment:
Test is too generic, since upstart and initv would be tested against this.
You could run something like [[ $(/sbin/init) == *"upstart"* ]] to make sure we have detected upstart.
Please test the above command line in your env to make sure it detects upstart